.packit.yaml

```diff
@@ -108,4 +107,4 @@ jobs:
       - packit
     dist_git_branches:
       - fedora-all
-      - epel-9
+      - epel-10-all
```

Comment:
We need to coordinate this with specfile and packit.

Reply:
There are no builds because the `epel-10-all` alias doesn't work yet.

Reply:
I will wait tomorrow, that fedora-distro-aliases should reach stable update, to test this PR properly and also push the packit one.